.github/CODEOWNERS

```diff
@@ -27,6 +27,8 @@
 /packages/post-purchase-ui-extensions-react/      @Shopify/checkout-extensibility
 /packages/retail-ui-extensions/			  @Shopify/retail-succeed-extensibility
 /packages/retail-ui-extensions-react/		  @Shopify/retail-succeed-extensibility
+/packages/pos-ui-extensions/			  @Shopify/retail-succeed-extensibility
```

Comment:
Let's update these to @Shopify/retail-succeed-build. Same goes for `retail-ui-extensions`.